src/Sulu/Component/Rest/ListBuilder/Expression/Doctrine/DoctrineWhereExpression.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\ListBuilder\Expression\Doctrine;
+
+use Doctrine\ORM\QueryBuilder;
+use Sulu\Component\Rest\ListBuilder\Doctrine\FieldDescriptor\AbstractDoctrineFieldDescriptor;
+use Sulu\Component\Rest\ListBuilder\Expression\WhereExpressionInterface;
+use Sulu\Component\Rest\ListBuilder\ListBuilderInterface;
+
+/**
+ * Represents a WHERE expression for doctrine - needs a field, a value and a comparator.
+ */
+class DoctrineWhereExpression extends AbstractDoctrineExpression implements WhereExpressionInterface
+{
+    /**
+     * Field descriptor used for comparison.
+     *
+     * @var AbstractDoctrineFieldDescriptor
+     */
+    protected $field;
+
+    /**
+     * Value which is used to compare.
+     *
+     * @var
```

Comment:
Same as above. I don't know the type because it could be any scalar type including null. All I can add is mixed.